src/app/userDashboard/registerMatches/page.tsx

```diff
@@ -163,108 +153,23 @@ const regMatches = () => {
-                          </span>
-                        </div>
-                      </div>
-                      <div className={styles.roomcreds}>
-                        <div className={styles.zeromin}>
-                          <span className={styles.roomId}>
-                            Room Password : {matchData?.password}
-                          </span>
-                        </div>
-                      </div>
-                    </div>
-                  ) : (
-                    <div className={styles.roomdetails_container}>
-                      <div className={styles.roomdetails}>
-                        <div className={styles.countdown}>
-                          <span className={styles.roomId}>
-                            Room Id : ********
-                          </span>
-                        </div>
-                      </div>
-                      <div className={styles.roomcreds}>
-                        <div className={styles.zeromin}>
-                          <span className={styles.roomId}>
-                            Room Password : ********
-                          </span>
-                        </div>
-                      </div>
-                    </div>
-                  )}
+                <span>Room Id: {visibleRooms?.find(room => room === matchData.roomUuid) ? matchData.roomId : '*****'}</span>
+                <span>Room password: {visibleRooms?.find(room => room === matchData.roomUuid) ? matchData.password : '*****'}</span>
                 </div>
               </div>
             </div>
           </div>
           <div>
 
           </div>
-          {/* use this code for your team member  */}
+          {/******  use this code for your team member  ****/}
 
           {/* <div className={styles.Teammembers}>Your Team Members</div>
-
-
-
           <div className={styles.container2}>
-
             <div className={styles.inner_cont}> */}
-
-          {/* <div key={index} className={`${styles.slide}`}> */}
-
-
-
-          {/* <div className={styles.reviewsContainer}>
-
-                <div className={styles.reviewCard}>
-
-                  <div className={styles.reviews}>
-
-                    <img
-
-                      src="/assests/reviewman.svg"
-
-                      alt="image"
-
-                      className={styles.profile}
-
-                    />
-
-                    <div className={styles.reviewer}>
-
-                      <div className={styles.name}>
-
-                        <h2>John doe</h2>
-
-                        <div className={styles.greenCircle}></div>
-
-                      </div>
-
-                      <p>akshay@gmail.com</p>
-
-                    </div>
-
-                  </div>
-                  <div  className={styles.review_close}>
-                  x
-                  </div>
-                </div>
-                
-              </div>
+              {/* <div key={index} className={`${styles.slide}`}> */}
```

Comment:
we donot need any comments

Reply:
this comment code use  for your team member section